TESTING-LISTMAKER.py

```diff
@@ -0,0 +1,37 @@
+# class Vendor:
```

Comment:
You can remove this from git, as it doesn't need to be in the submission.

`git rm TESTING-LISTMAKER.py` (and then commit).